src/documentation/common/CodeEmbed.tsx

```diff
@@ -109,6 +122,15 @@ const CodeEmbed = ({ code: codeWithImports, parentSlug }: CodeEmbedProps) => {
           ref={codeRef}
           background={state === "default" ? "white" : "blimpTeal.50"}
           highlightDragHandle={state === "raised"}
+          tabIndex={0}
+          _focus={{
+            boxShadow: "var(--chakra-shadows-outline);",
+          }}
+          _focusVisible={{
+            outline: "none",
+          }}
+          onKeyDown={handleKeyDown}
+          zIndex={1}
```

Comment:
I blame this for https://github.com/microbit-foundation/python-editor-next/issues/727